### ISSUE TITLE
Local setup issues

### ISSUE THREAD

**Issue by MrJithil** [storybookjs/frontpage#600]
I am building the storybook frontpage in my local machine, but its failing due to `Error: Source GraphQL API: HTTP error 401 Unauthorized` 

Someone please guide me the local environment `.env` file or the way we are using to generate the api keys here?

**Comment by winkerVSbecks** [comment]
That's not public info. You can use `yarn start:docs-only` to run the docs pages locally.

**Comment by LuLaValva** [comment]
I am having the same issue as @MrJithil, but it also happens with `:docs-only`. I have no modifications to the repo, just a fresh clone and a `yarn install`.

**Comment by winkerVSbecks** [comment]
@LuLaValva can you drop by the Storybook discord. We have a channel dedicated to contributing. Happy to help you there.

**Comment by MrJithil** [comment]
> I am having the same issue as @MrJithil, but it also happens with `:docs-only`. I have no modifications to the repo, just a fresh clone and a `yarn install`.

Yeah. It's not working as per the local setup instructions. I have notified @kylegach regarding this in Discord, and he agreed to look into this once he got some free time.

**Comment by kylegach** [comment]
Sorry folks. I'll be looking into this sometime this week.

**Comment by MrJithil** [comment]
Any progress?
